gen/container_test.go

```diff
@@ -32,13 +34,13 @@ import (
 func TestCollectionsOfPrimitives(t *testing.T) {
 	tests := []struct {
 		desc string
-		p    testdata.PrimitiveContainers
+		p    containers.PrimitiveContainers
```

Comment:
is it worth trying to reduce the stutter here? Maybe by importing the testdata subpackages as like "tc" "te" "ts"